### DISCUSSION TITLE
An error is reported when querying the endpoint

### DISCUSSION THREAD

**Discussion by 1450793561** [apache/skywalking#11626] [Q&A]
skywalking version：9.6.0

<img width="1679" alt="22222222" src="https://github.com/apache/skywalking/assets/31468018/8be916bd-deff-4378-8ae6-877875b5fa4b">

<img width="1239" alt="111111" src="https://github.com/apache/skywalking/assets/31468018/b227480f-6291-4cc4-8588-e9c08b7eb074">
<img width="1204" alt="企业微信截图_29c5da2e-14bf-495f-9c40-dc70cd057ed8" src="https://github.com/apache/skywalking/assets/31468018/0297059a-05cf-48b9-8cd3-2049127f5812">

`
2023-12-04 11:09:51,884 - notprivacysafe.graphql.GraphQL - 501 [armeria-eventloop-epoll-5-4] WARN  [] - Query did not parse : 'query queryData($duration: Duration!,$entity0: Entity!,$e
xpression00: String!,$subExpression00: String!,$expression01: String!,$subExpression01: String!,$expression02: String!,$subExpression02: String!,$entity1: Entity!,$expression10: String
!,$subExpression10: String!,$expression11: String!,$subExpression11: String!,$expression12: String!,$subExpression12: String!,$entity2: Entity!,$expression20: String!,$subExpression20:
 String!,$expression21: String!,$subExpression21: String!,$expression22: String!,$subExpression22: String!,$entity3: Entity!,$expression30: String!,$subExpression30: String!,$expressio
n31: String!,$subExpression31: String!,$expression32: String!,$subExpression32: String!,$entity4: Entity!,$expression40: String!,$subExpression40: String!,$expression41: String!,$subEx
pression41: String!,$expression42: String!,$subExpression42: String!,$entity5: Entity!,$expression50: String!,$subExpression50: String!,$expression51: String!,$subExpression51: String!
,$expression52: String!,$subExpression52: String!,$entity6: Entity!,$expression60: String!,$subExpression60: String!,$expression61: String!,$subExpression61: String!,$expression62: Str
ing!,$subExpression62: String!,$entity7: Entity!,$expression70: String!,$subExpression70: String!,$expression71: String!,$subExpression71: String!,$expression72: String!,$subExpression
72: String!,$entity8: Entity!,$expression80: String!,$subExpression80: String!,$expression81: String!,$subExpression81: String!,$expression82: String!,$subExpression82: String!,$entity
9: Entity!,$expression90: String!,
`

**Comment by 1450793561** [comment]
How to resolve this？I don‘t find args in config/application.yaml

**Comment by heyanlong** [comment]
I think because new limits have been introduced to GraphQL.
You can use `graphql.parser.ParserOptions;` to set a higher max tokens value.

**Reply by 1450793561** [reply]
This is config in oap?I can't find this

oap server config SW_QUERY_MAX_QUERY_COMPLEXITY is 30000
```
query:
  selector: ${SW_QUERY:graphql}
  graphql:
    # Enable the log testing API to test the LAL.
    # NOTE: This API evaluates untrusted code on the OAP server.
    # A malicious script can do significant damage (steal keys and secrets, remove files and directories, install malware, etc).
    # As such, please enable this API only when you completely trust your users.
    enableLogTestTool: ${SW_QUERY_GRAPHQL_ENABLE_LOG_TEST_TOOL:false}
    # Maximum complexity allowed for the GraphQL query that can be used to
    # abort a query if the total number of data fields queried exceeds the defined threshold.
    maxQueryComplexity: ${SW_QUERY_MAX_QUERY_COMPLEXITY:30000}
    # Allow user add, disable and update UI template
    enableUpdateUITemplate: ${SW_ENABLE_UPDATE_UI_TEMPLATE:false}
    # "On demand log" allows users to fetch Pod containers' log in real time,
    # because this might expose secrets in the logs (if any), users need
    # to enable this manually, and add permissions to OAP cluster role.
    enableOnDemandPodLog: ${SW_ENABLE_ON_DEMAND_POD_LOG:false}
```

**Reply by 1450793561** [reply]
Where is   this `graphql.parser.ParserOptions` ?
